From 36af5dff831900abaa38c6c906a2c9ac8ca4d201 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 23 Feb 2016 18:24:50 +0100
Subject: [PATCH] iplink: update available type list

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1269528
Upstream Status: iproute2.git commit 1253a10a63e41

commit 1253a10a63e417c5a7774c127aea1a3b0dd09e88
Author: Nicolas Dichtel <nicolas.dichtel@6wind.com>
Date:   Tue Oct 8 07:59:45 2013 -0700

    iplink: update available type list

    macvtap and vti were missing.

    Signed-off-by: Nicolas Dichtel <nicolas.dichtel@6wind.com>
---
 ip/iplink.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ip/iplink.c b/ip/iplink.c
index 048d4f3..5a9ed57 100644
--- a/ip/iplink.c
+++ b/ip/iplink.c
@@ -89,9 +89,9 @@ void iplink_usage(void)
 	if (iplink_have_newlink()) {
 		fprintf(stderr, "       ip link help [ TYPE ]\n");
 		fprintf(stderr, "\n");
-		fprintf(stderr, "TYPE := { vlan | veth | vcan | dummy | ifb | macvlan | can |\n");
-		fprintf(stderr, "          bridge | ipoib | ip6tnl | ipip | sit | vxlan |\n");
-		fprintf(stderr, "          gre | gretap | ip6gre | ip6gretap }\n");
+		fprintf(stderr, "TYPE := { vlan | veth | vcan | dummy | ifb | macvlan | macvtap |\n");
+		fprintf(stderr, "          can | bridge | ipoib | ip6tnl | ipip | sit | vxlan |\n");
+		fprintf(stderr, "          gre | gretap | ip6gre | ip6gretap | vti }\n");
 	}
 	exit(-1);
 }
-- 
1.8.3.1

